send_message.py

```
@@ -65,214 +93,104 @@ def parse_hl7_file(filepath: PosixPath) -> str:
-    for i in range(8, 18, 1):
-        for day in ["monday", "tuesday", "wednesday", "thursday", "friday"]:
-            getattr(schedule.every(), day).at(f"{i:02d}:00").do(
-                parse_and_send_message, epic_socket, paths, test, host, port
-            )
+    # handle connect and sending of JSON data to local server
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+        try:
+            s.connect(("127.0.0.1", port))
+        except Exception as e:
+            logger.exception(f"Failed connecting to 127.0.0.1:{port}")
+            raise e
```

Ruff (0.14.7)

Use `raise` without specifying exception name

Remove exception name

(TRY201)